components/mpas-framework/Makefile

```diff
@@ -296,6 +296,12 @@ intel-mpi:
 	"CPPFLAGS = $(MODEL_FORMULATION) -D_MPI" )
 
 gfortran:
+	GFORTRAN_GTE_10=$$(expr `gfortran -dumpversion | cut -f1 -d.` \>= 10) ;\
+	if [ "$${GFORTRAN_GTE_10}" = "1" ]; then \
```

Comment:
Should this be single dollar sign?  My understanding was that $$ accesses shell variables, but single dollar sign should be used for Makefile variables (https://stackoverflow.com/questions/26564825/what-is-the-meaning-of-a-double-dollar-sign-in-bash-makefile).  If my understanding is correct, it is interesting that using $$ here still works.

The $$ is also used below on all the lines below.

Reply:
Huh. I tested the makefile but checking for version `>=1` so it adds the flag. Compile dies with an error with gnu V6 (as expected) because the flag `-fallow-argument-mismatch` is undefined.  I tried to change the double dollar signs to `$(GFORTRAN_GTE_10)` and `$(EXTRA_FFLAGS)` and it didn't work (the variables don't contain the correct values). So @xylar did it right - I just don't understand why these are shell variables rather than makefile variables.

Reply:
Ok, thanks for the explanation and testing.  I'm convinced. :)